lib/linter-reek.js

```diff
@@ -19,6 +16,41 @@ const loadDeps = () => {
   }
 };
 
+const parseReekSyntaxError = (error, filePath) => {
+  const exceptionMessage = /Exception message:\s*(.*)/g.exec(error);
+  return [{
+    severity: 'error',
+    excerpt: exceptionMessage ? `linter-reek: ${exceptionMessage[1]}` : 'linter-reek: unexpected error',
+    location: {
+      file: filePath,
+      // first line of the file
+      position: [[0, 0], [0, Infinity]],
```

Comment:
Suggestion: If you pass `editor` in here, you can use `helpers.generateRange(editor, 0)` to highlight just the _content_ of the first line.

This usually makes no difference though, since it's rare for the first line to be indented, so don't bother changing this unless you want to.